uarray/_uarray_dispatch.cxx

```diff
@@ -199,24 +193,34 @@ PyObject * clear_all_globals(PyObject * /*self*/, PyObject * /*args*/)
 }
 
 
-PyObject * set_global_backend(PyObject * /*self*/, PyObject * args)
+PyObject * set_global_backend(PyObject * Py_UNUSED(self), PyObject * args)
 {
   PyObject * backend;
-  if (!PyArg_ParseTuple(args, "O", &backend))
+  int only = false, coerce = false;
+  if (!PyArg_ParseTuple(args, "O|pp",
+    &backend,
+    &coerce,
+    &only))
     return nullptr;
 
   auto domain = backend_to_domain_string(backend);
   if (domain.empty())
     return nullptr;
 
-  global_domain_map[domain].global = py_ref::ref(backend);
+  backend_options options;
+  options.backend = py_ref::ref(backend);
+  options.coerce = coerce;
+  options.only = only;
+
+  global_domain_map[domain].global = options;
   Py_RETURN_NONE;
 }
 
-PyObject * register_backend(PyObject * /*self*/, PyObject * args)
+PyObject * register_backend(PyObject * Py_UNUSED(self), PyObject * args)
```

Comment:
`Py_UNUSED` is _maybe_ unused, commenting out the name is compile error on use.

Reply:
IIRC this also modifies the name so using it is a compiler error.

Reply:
https://github.com/python/cpython/blob/b49858b4b7b4c9d85ef6946ad020f83e4fa1caa7/Include/pymacro.h#L98

It adds `__attribute__((unused))` which as per GCC's docs
>This attribute, attached to a variable, means that the variable is meant to be **possibly unused**. GCC will not produce a warning for this variable. 

(Emphasis mine)

Reply:
Actually, it does modify the name...
Still no reason to use it in C++.